blog: December 2016 newsletter

xprazak2 opened the pull request:
Incomplete draft of this year's last newsletter. If you have anything that should be there, just let me know.

Commit:
blog: December 2016 newsletter

GregSutcliffe commented on _posts/2016-12-30-foreman-community-newsletter---december-2016.md:
```diff
@@ -0,0 +1,64 @@
+  in the very last Foreman blog post of 2016.
+---
+
+### Foreman 1.14 branched
```
RC1 is out now - and it's probably better to link to that announcement, since it has highlights etc: https://groups.google.com/d/topic/foreman-announce/Y2yImbWecw0/discussion

GregSutcliffe replied:
Putting some of the highlights here is probably good too, as is a call for testing :)

dLobatog replied:
There have been 2 people complaining about wrong signatures on IRC - test it on your own and add a comment if it doesn't work. centos-release-scl has a different signature to other packages in there

GregSutcliffe commented on _posts/2016-12-30-foreman-community-newsletter---december-2016.md:
```diff
@@ -0,0 +1,64 @@
+### Katello 3.3 release
+
+
+Katello 3.3 was released on the ??th Dec, [announcement][k3_3].
```
I believe branching is imminent, @ehelms?

dLobatog replied:
??th?

GregSutcliffe commented on _posts/2016-12-30-foreman-community-newsletter---december-2016.md:
```diff
@@ -0,0 +1,64 @@
+
+### New media and blogs
+
+TODO: None yet!
```
Obviously just delete the section if its empty by the end :P

GregSutcliffe commented on _posts/2016-12-30-foreman-community-newsletter---december-2016.md:
```diff
@@ -0,0 +1,64 @@
+
+The latest instalment of the community demo series showcases all that's
+happening in the Foreman community. In the last demo we covered UI background tasks, ignoring facts on import,
+new Ansible options and more!
```
Make a big point of the panel at the end, I think it was the best part of this recording, and may spark some discussion if we're lucky :)

GregSutcliffe commented on _posts/2016-12-30-foreman-community-newsletter---december-2016.md:
```diff
@@ -0,0 +1,64 @@
+
+### Upcoming events
+
+_(All Foreman events are now available via the [Foreman Calendar](/events))_
```
Conferences - devconf/fosdem/cfgmgmt should be promoted.

Our hackday was published this morning, see https://groups.google.com/d/topic/foreman-users/QYIHyJ1kSBc/discussion and https://foreman-construction-day-2017.eventbrite.com/

GregSutcliffe commented on _posts/2016-12-30-foreman-community-newsletter---december-2016.md:
```diff
@@ -0,0 +1,64 @@
+
+- [foreman_ansible](https://github.com/theforeman/foreman_ansible) updated to 1.3.0
+- [foreman_bootdisk](https://github.com/theforeman/foreman_bootdisk) updated to 8.1.0
+
```
I also see foreman_remote_execution bumped to 1.2.2

GregSutcliffe reviewed (changes requested):
Good start, there's more to add though. I'll have a think over the weekend if there's anything else and we can merge next week. Thanks!

xprazak2 commented:
Updated to address comments:

* added rex version update
* pointing to February events explicitely
* pointing to 1.14-RC1 instead of 1.14
* stressing the panel discussion on the last demo

Waiting for Katello release.

GregSutcliffe commented on _posts/2016-12-30-foreman-community-newsletter---december-2016.md:
```diff
@@ -0,0 +1,83 @@
+The latest instalment of the community demo series showcases all that's
+happening in the Foreman community. In the last demo we covered UI background tasks, ignoring facts on import and
+new Ansible options. Second part of demo recording contains a panel discussion of Foreman team members. You can hear
+the highlights of 2016 and recapitulation the last 12 months with Foreman,
```
I'd just put this as "and recap of the last 12 months" - "recapitulation" is technically correct but sounds too formal :)

GregSutcliffe commented on _posts/2016-12-30-foreman-community-newsletter---december-2016.md:
```diff
@@ -0,0 +1,83 @@
+
+### Upcoming events
+
+We will be at [DevConf][devconf], [FOSDEM][fosdem], and [CfgMgmtCamp][cfgmgmc]. There will be [Foreman Construction Day][fcd] on 8th Feb in Ghent, Belgium. Do not forget to [register][fcd_register] if you are interested.
```
"There will be *a* ..." or "There will be *another*" - you need something after the "be" to make the sentence flow.

GregSutcliffe commented on _posts/2016-12-30-foreman-community-newsletter---december-2016.md:
```diff
@@ -0,0 +1,83 @@
+[fosdem]: https://fosdem.org)
+[cfgmgmc]: http://cfgmgmtcamp.eu
+
+[Foreman]
```
Leftover?

GregSutcliffe commented on _posts/2016-12-30-foreman-community-newsletter---december-2016.md:
```diff
@@ -0,0 +1,83 @@
+_(All Foreman events are now available via the [Foreman Calendar](/events))_
+
+[fcd]: https://groups.google.com/forum/#!topic/foreman-users/QYIHyJ1kSBc/discussion
+[fcd_register]: https://www.eventbrite.com/e/foreman-construction-day-registration-30351669661
```
correct, but you can also use https://foreman-construction-day-2017.eventbrite.com/ if you prefer

GregSutcliffe commented:
Looks 99% there, just a few nitpicks. @ehelms any word on the release, or shall we publish without it?

tbrisker commented on _posts/2016-12-30-foreman-community-newsletter---december-2016.md:
```diff
@@ -0,0 +1,83 @@
+[fcd]: https://groups.google.com/forum/#!topic/foreman-users/QYIHyJ1kSBc/discussion
+[fcd_register]: https://www.eventbrite.com/e/foreman-construction-day-registration-30351669661
+[devconf]: https://devconf.cz
+[fosdem]: https://fosdem.org)
```
extra )

xprazak2 commented:
Updated with changes, waiting for Katello release.

xprazak2 commented:
Added foreman_discovery to updated plugins

mmoll commented:
IMHO we should get this out now, after "Katello 3.3" -> "Katello "3.2.1"

xprazak2 commented:
I was waiting for an announcement on mailing list so I could reference it - same as for 3.2.1. I also wasn't sure which features in 3.3 to highlight, so I just kept it very brief.

Edit: I removed Katello 3.3 since there hasn't been an announcement yet.

mmoll commented:
merged as d656733e with the Katello version in the excerpt fixed, díky @xprazak2!
